src/settings/toml/tests/deploy_target.rs

```diff
@@ -31,8 +48,11 @@ fn it_errors_on_zoneless_deploy_target_missing_workers_dev() {
 }
 
 #[test]
-fn it_errors_on_deploy_target_missing_name() {
-    let test_toml = WranglerToml::webpack_zoneless("", true);
+fn it_errors_on_deploy_target_missing_account_id() {
+    let script = "zoneless_no_account_id";
+    let account_id = "account_id";
+    let mut test_toml = WranglerToml::webpack_zoneless(script, account_id, true);
+    test_toml.account_id = None;
```

Comment:
can we pass None directly to `webpack_zoneless`?

Reply:
discussed in air; the idea is to have convenience constructors for valid tomls, and manipulate them manually in the case that we want to build "EVIL" tomls